ui/app/controllers/jobs.js

```diff
@@ -13,26 +11,4 @@ export default Controller.extend({
   isForbidden: false,
 
   jobNamespace: 'default',
-
-  // The namespace query param should act as an alias to the system active namespace.
-  // But query param defaults can't be CPs: https://github.com/emberjs/ember.js/issues/9819
-  syncNamespaceService: forwardNamespace('jobNamespace', 'system.activeNamespace'),
-  syncNamespaceParam: forwardNamespace('system.activeNamespace', 'jobNamespace'),
 });
-
-function forwardNamespace(source, destination) {
-  return observer(source, `${source}.id`, function() {
```

Comment:
It didn't make it all the way through the branch, but I just wanted to throw out a 👍 for the `afterSetup` route hook approach you originally took to replace this observer:

> 👍

Reply:
Which one was that? I have already suppressed all memories of this code 😅